scripts/jest/visual-adapter.js

```diff
@@ -34,11 +45,13 @@ const BoxVisualTestUtils = {
     },
 
     // Takes image screenshots after user input, e.g., clicking or entering text
-    takeScreenshotAfterInput: async (id, selector, action = 'click', userInput) => {
+    takeScreenshotAfterInput: async (id, selector, action = 'click', userInput, afterInputSelector = null) => {
         await BoxVisualTestUtils.gotoStory(id);
-        await global.page.waitForSelector(selector);
+        await global.page.waitForSelector(selector, { visible: true });
         await global.page[action](selector, userInput);
-        await global.page.waitFor(100);
+        (await afterInputSelector) !== null
```

Comment:
Do we need to strict check `null`? I.e. should it just check for falsey values in case we pass in undefined?